### ISSUE TITLE
Mounting crashes on minecraft server but not on singleplayer

### ISSUE THREAD

**Issue by WaterGenic** [darkevilmac/archimedes-ships-plus#362]
When I use the mod on my server it crashes my whole minecraft and when I use it on singleplayer it doesn't crash at all. It has the same mods as my client. When I assemble it's okay but when I mount it crashes. 

Crash Report: https://docs.google.com/document/d/1uBbnJ5Mi6J9AwGPq9i390RH6Hi7mQinNpQUIHu6szhU/edit?usp=sharing

**Comment by twarren23** [comment]
Yeah I'm having this exact problem too. I haven't figured it out yet.

**Comment by BlueOpRed** [comment]
me too

i have a solution try using a crash destroying client for minecraft

or try twitch minecraft laucher

**Comment by Builderdog841** [comment]
If you install an older version it works
